[Detection Engine][Exceptions] - Update docs for single and shared exceptions (#4021)
elastic/security-docs: docs/detections/api/exceptions/api-update-exception-container.asciidoc

exception containers. |No
|`type` |String a|The type of exception, which must be one of these:

> |`type` |String a|The type of exception list, which must be one of these:

Reviewer: ```suggestion
|`type` |String a|The type of exception list, which must be one of:
```

Reviewer: You could also go with "Valid values are:" which appears elsewhere in this API

Author: I like this ^. It's much more explicit. 

```suggestion
|`type` |String a|The type of exception list. Valid values are:
```
